### DISCUSSION TITLE
config/global

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#65] [Announcements]
# config/global

详细介绍主题提供的全局能力

http://localhost:5173/config/global.html

**Comment by 1272171893** [comment]
没有提供baseURL 这个选项吗？

**Reply by ATQQ** [reply]
支持的

继承官方配置的，这里只列了主题拓展的

**Reply by ATQQ** [reply]
https://vitepress.dev/reference/site-config#base

**Reply by ATQQ** [reply]
![](https://img.cdn.sugarat.top/mdImg/MTY3ODYzMDgyMjY4MQ==678630822681)

**Comment by jackloves111** [comment]
我只想要在原主题的基础上增加一个单独的公告弹窗，请问怎么引入呢？

**Reply by ATQQ** [reply]
https://theme.sugarat.top/config/global.html#blog


就这个文档里面有个blog属性 设置为false即可

**Comment by changyongyong** [comment]
footer底部申明中的@sugarat/theme@0.3.6咋修改，没看到配置

**Reply by ATQQ** [reply]
这个版本是自动生成的是你当时用的主题包版本，只提供了配置项目隐藏

https://theme.sugarat.top/config/global.html#footer

里面有隐藏的例子

**Comment by chay73** [comment]
求就算是标准vitepress 模板也可以用的，在文章页面显示字数和阅读时间的教程，真的很需要这个，谢谢🙏

**Reply by ATQQ** [reply]
https://theme.sugarat.top/config/global.html#blog

继承主题关闭博客模式就行。

参考这个文档站配置：https://github.com/ATQQ/easypicker2-client/blob/main/docs/.vitepress/config.mts

**Comment by sunset2131** [comment]
在原主题如何实现暗黑模式扩散动画呢？
